app/components/PluginDetail.jsx

```diff
@@ -176,6 +176,11 @@ class PluginDetail extends React.PureComponent {
                 </div>
                 <h5>Labels</h5>
                 {this.getLabels(plugin.labels)}
+                
+                <div className="update-link">
+                <h6>Are you maintaining this plugin?</h6>
+                <p>Visit the <a href={plugin.wiki.url} target="_wiki">Jenkins Plugin Wiki</a> to edit this content.</p>
```

Comment:
It seems all wiki URLs are harmless so can probably leave this as is.

These are the only ones outside `a-zA-Z0-9._+-`:

https://wiki.jenkins-ci.org/display/JENKINS/Centralized+Job%28Re%29Action+Plugin
https://wiki.jenkins-ci.org/display/JENKINS/Compuware+Source+Code+Download+for+Endevor%2C+PDS%2C+and+ISPW+Plugin
https://wiki.jenkins-ci.org/display/JENKINS/Dead+Man%27s+Snitch+Plugin
https://wiki.jenkins-ci.org/display/JENKINS/Hudson+Speaks%21+Plugin
https://wiki.jenkins-ci.org/display/JENKINS/Mobile+App+Distribution+%28MDT%29++Plugin
https://wiki.jenkins-ci.org/display/JENKINS/Plugin+Usage+Plugin+%28Community%29

And plugins with exceptionally weird page titles get a completely different URL that's safe to not quote.

https://wiki.jenkins-ci.org/pages/viewpage.action?pageId=60915753
https://wiki.jenkins-ci.org/pages/viewpage.action?pageId=99910492
https://wiki.jenkins-ci.org/pages/viewpage.action?pageId=99910545